audiomentations/augmentations/duration_clip.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+
+
+class DurationClip(BaseWaveformTransform):
```

Comment:
I would prefer RandomCrop over DurationClip, because the term Clip is can confused with the [`Clip` transform](https://iver56.github.io/audiomentations/waveform_transforms/clip/) that we already have, which relates to clipping distortion. A clip can also refer to a small segment of audio that is extracted from a larger file.

I'm not 100% sure about the name RandomCrop though, because cropping is commonly used about images/rasters (e.g. [like this in albumentations](https://albumentations.ai/docs/api_reference/augmentations/crops/transforms/)), but I guess it _could_ sometimes be used about audio too. Some industry people like to say trim or cut. Maybe CutToLength could work? Or TrimPadToLength? 🤔 @Cangonin @mmxgn Do you have any thoughts/opinions on this?

```suggestion
class RandomCrop(BaseWaveformTransform):
```

Reply:
Isn't this functionally the same as a TimeTrim transform just implemented differently?

Reply:
I don't know. Is that an established term?
![bilde](https://user-images.githubusercontent.com/1470603/227871390-3af096cd-103c-41c4-b403-568386c575fa.png)

Reply:
How about `AudioLengthAdjust`?

Reply:
Or AdjustAudioLength? Or AdjustLength or LengthAdjust, since all audiomentations transforms work on audio anyway? 🤷

Naming sure is hard xD

Reply:
I agree that clipping refers more to distortion, that's how I interpreted it when I saw the name of this transform. Since the audio can be trimmed or padded after this transform, I like your suggestion `AdjustDuration` :)

Reply:
I also like `AdjustDuration` too.

Reply:
Does it sound like "adjusting the duration" is a time stretching operation though? 🤔

Reply:
I see your point with `AdjustDuration`. I think that `AdjustLength` works too.

Reply:
AI gives a vote to AdjustDuration

![bilde](https://user-images.githubusercontent.com/1470603/227892294-f6adc086-16e4-40aa-a5fa-7477340c6a41.png)

Reply:
What about `Resize` ?

Reply:
The term `Resize` is common in image processing, but that implies some sort of resampling, which is not what we are doing here. I think we'll roll with AdjustDuration, since that seems to have the most votes now